src/Classes/ServiceAPI/MyRadio_User.php

```diff
@@ -1371,8 +1406,22 @@ public function getEditForm()
                         ['value' => 'f', 'text' => 'Female'],
                         ['value' => 'o', 'text' => 'Other']
                     ]
-                ]))
-                ->addField(new MyRadioFormField('sec_personal_close', MyRadioFormField::TYPE_SECTION_CLOSE));
+                ]));
+        if (empty(Config::$contract_uri) === false) {
```

Comment:
if (!empty), although thinking about it, that's personal preference, so ignore me.